API.md

```diff
@@ -74,9 +74,9 @@ Filename extensions of user-editable documents.
 ### `history`
 History of edits done to `doc`.
 
-### `editCount`
+### `docChangeCounter`
 
-Index of the current log entry in the `history`. Is incremented with every new edit and updated on undo/redo. 
+Keeps track of changes to the `doc`, including undo and redo, and indicates a changed `doc`. 
```

Comment:
```suggestion
A change in the property indicates a change to the `doc` as a result of edits (including undo and redo). 
```

I think we should focus on the change and not the content of the property. It doesn't, by itself track the changes to the `doc`, it just changes when the `doc` changes and that is all we care about.